tools/.gitignore

```diff
@@ -3,3 +3,4 @@
 /downloads/
 
 vendor/
+/src/py-mkopensource/py-mkopensource
```

Comment:
This doesn't need to be here.

Reply:
@LukeShu @kflynn I added the exclude since while I'm testing I compile the app and didn't want to commit it by mistake. 
Why do you think this exclude should not be here? Is there a guideline about what should/shouldn't be in .gitignore files?

Reply:
In normal usage, this will be built in `tools/bin/py-mkopensource`, rather than anywhere else...